includes/class.llms.course.data.php

```diff
@@ -164,6 +164,12 @@ public function get_lesson_completions( $period = 'current' ) {
 
 		$lessons = implode( ',', $this->post->get_lessons( 'ids' ) );
 		// phpcs:disable WordPress.DB.PreparedSQL.InterpolatedNotPrepared
+
+		// Return early for courses without any lessons.
+		if(empty($lessons)){
```

Comment:
Use [WordPress coding standards for spaces](https://developer.wordpress.org/coding-standards/wordpress-coding-standards/php/#space-usage).
```suggestion
		if ( empty( $lessons ) ) {
```